src/app/pages/videos/videos-list/videos-container/videos-container.component.html

```diff
@@ -0,0 +1,2 @@
+<app-videos-header></app-videos-header>
+<app-videos-list-container></app-videos-list-container>
```

Comment:
This is a use less component. It is adding no value. Its not used for layout or data fetching. I think we should remove this

Reply:
I agree. will do some changes